app/src/main/java/ch/epfl/sdp/entity/PlayerManager.java

```diff
@@ -15,14 +15,12 @@
  * beginning of each game and all players should be removed at the end of each game.
  */
 public class PlayerManager {
-    /**
-     * The maximum number of player in a lobby
-     */
-    public static final int NUMBER_OF_PLAYERS_IN_LOBBY = 2;
 
     /**
      * All the names of the firebase collections
      */
+    public static final int NUMBER_OF_PLAYERS_IN_LOBBY = 2;
```

Comment:
This is not the name of the firebase collection, just the maximum number of players in the lobbt

Reply:
I screwed up a copy paste and this comment appeared out of nowehere. Thanks for pointing it out though